.github/workflows/ci.yml

```diff
@@ -110,7 +109,7 @@ jobs:
           if [ ${{ runner.os }} == 'macOS' ]; then
               export MACOSX_DEPLOYMENT_TARGET=10.9;
           fi
-          python setup.py ${{ matrix.BUILD_COMMAND }}
```

Comment:
Using `--no-isolation` to spare creating a virtual env and to use pre-release versions of `numpy` and `cython`